include/subjects.json

```diff
@@ -859,4 +859,11 @@
         "SCI AND TECH. RUBBER & RUBBER LIKE",
         false
     ]
+    "MA20101": [
+        "TRANSFORM CALCULUS"
```

Comment:
Also add a comma and line with `false`. `false` indicates if we have the paper or not. Do same thing for the next subject code.